compilers/concrete-compiler/compiler/include/concretelang/Support/Pipeline.h

```diff
@@ -96,8 +96,7 @@ mlir::LogicalResult extractSDFGOps(mlir::MLIRContext &context,
 
 mlir::LogicalResult
 lowerConcreteToStd(mlir::MLIRContext &context, mlir::ModuleOp &module,
-                   std::function<bool(mlir::Pass *)> enablePass,
-                   bool simulation);
+                   std::function<bool(mlir::Pass *)> enablePass);
```

Comment:
Were there any other changes that support the runtime context during simulation?

Reply:
Nope. I discussed that with Quentin, and we thought that we would be better off avoiding to maintain two calling conventions for a negligeable runtime improvement (allocating empty buffers for the runtime context). Plus this made the refactoring simpler.